### DISCUSSION TITLE
Multiple RTSP Streams

### DISCUSSION THREAD

**Discussion by bbrodriguez** [tphakala/birdnet-go#181] [Ideas]
I have a few outdoor IP cameras and have RTSP streams available for them all.

I'd love to have multiple RTSP stream inputs.

I tried to fudge ffmeg with:

```
rtsp:
  url: "rtsp://xx.xx.xx.xx:8554/back -i rtsp://xx.xx.xx.xx:8554/front" # RTSP stream URL
  transport: tcp # RTSP Transport Protocol
```

But that failed.

**Comment by tphakala** [comment]
It is not yet supported, there is issue tracking this feature https://github.com/tphakala/birdnet-go/issues/107

**Reply by tphakala** [reply]
Support for multiple RTSP streams is now implemented and available in dev images, please see https://github.com/tphakala/birdnet-go/discussions/187 for more information
